### PR TITLE
Check it out. It's a ghetto first pass but it works!

### PR DESCRIPTION
**Pull request by azethoth**
first pass at youtube player. no styling, no search, no validation, and it only accepts single video urls

i'll work on search next so you don't have to cut and paste a url. i didn't bother to style it because the application view is gonna change.

### CODE REVIEW

File: views/styles.sass [hunk 1]
```diff
@@ -141,6 +141,13 @@ span.pictos
       &:hover
         background: #111 url(/images/google_32.png) no-repeat 8px center
 
+#ytplayer_wrapper
+  height: 100%
+  width: 100%
+
+#ytplayer_wrapper.stowed
```
Comment (azethoth) [inline views/styles.sass]: ahh just realized i can probably nest this with sass no?

File: app.rb [hunk 2]
```diff
@@ -104,6 +106,21 @@ class Song < Ohm::Model
     })
   end
 
+  # TODO fix this up to use api to get video metadata
+  def self.from_youtube_url(youtube_url)
+    query = Addressable::URI::parse(youtube_url).query_values
+    Song.new({
+      source:     'youtube',
+      title:      'YouTube Video',
```
Comment (azethoth) [inline app.rb]: when i code up a proper youtube search i can get all the video metadata from the api so i'll have all these attributes for real, even a thumbnail

**Comment by arbales** [comment]
Neato — now I can play the theme song from Sesame Street each and every morning.

File: views/styles.sass (hunk 1 above)
Comment (nixme) [inline views/styles.sass]: yah, indent and use `&.stowed`  (I think, since it's not a child selector but rather multiple selectors together)

**Comment by nixme** [comment]
Awesome! I'm finishing up the rails migration and then will merge that to master then merge this in.

**Comment by arbales** [comment]
noooooooooooooooooooooooooooooooooooooo, sinatraaaaaaaaaaaaaaaaaaaaaaa

**Comment by azethoth** [comment]
i pushed a new version that does search and click to add songs. still ghetto, didn't bother styling it yet and only 1 page of search results come back but at least the video metadata in the song queue is right now.

**Comment by azethoth** [comment]
i'll give it some basic styling tomorrow and probably get the results paging in so we can use it on monday

**Comment by azethoth** [comment]
bah, there are a bunch of bugs in the search view. working on it

**Comment by azethoth** [comment]
paging added. i'm all done with youtube for now. feel free to change up the styling if you guys like

**Comment by nixme** [comment]
Okay, finally finished the rails migration and also broke up all the javascript and moved templates into their own files. I'm going to start to merge this.

**Pull request merged**